main.rb

```diff
@@ -73,6 +97,25 @@ class Saying
   end
 end
 
+put '/send-to-slack' do
+  saying = Saying.find(params[:id])
+  if saying.inc(:crap_count, 1)
+    @attachments = [
+      {
+        author_name: saying.who,
+        color: 'good',
+        title: 'saywut!',
+        text: saying.wut
+      }
+    ]
+    Slack::Post.post_with_attachments saying.who,
+    @attachments, '#lemon-party-room'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.